notebooks/README.md

```diff
@@ -0,0 +1,7 @@
+# ltu-ili tutorials
+
+This directory contains jupyter notebook tutorials for the ltu-ili package.
+
+* [tutorial.ipynb](./tutorial.ipynb) - A tutorial for using the `sbi` backend, including NPE, NLE, and use of CNN embedding networks.
```

Comment:
The tutorial notebook has been renamed "sbi", so please change this in the README.

Reply:
Done.